code/modules/mob/living/death.dm

```diff
@@ -49,8 +49,14 @@
 	timeofdeath = world.time
 	tod = worldtime2text()
 	var/turf/T = get_turf(src)
+	var/area/A = get_area(T)
+	if(src.mind.special_role == "Cult Master")
+		for(var/datum/mind/M in SSticker.mode.cult)
+			if(istype(M.current, /mob/living) && isliving(M.current))
+				M.current << "<span class='cultlarge'> Your cult's master, [mind.name], has fallen in the [A.name]!"
+			else
+				continue

```

Comment:
I made this PR shortly after I returned and my coding was even more atrocious than it is now.